core/src/main/java/io/grpc/Grpc.java

```diff
@@ -40,4 +40,11 @@ private Grpc() {
   @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1710")
   public static final Attributes.Key<SSLSession> TRANSPORT_ATTR_SSL_SESSION =
           Attributes.Key.of("ssl-session");
+
+  //TODO(zhangkun83): create tracking issue
+  @ExperimentalApi
+  public enum Side {
```

Comment:
This should probably exist in the InternalMethodDescriptor class.

Reply:
MethodDescriptor can't reference InternalMethodDescriptor. Since generateTraceSpanName() is no longer public, it's not necessary to make it good-looking. I deleted the enum and switched to boolean instead.